impeller/aiks/paint.h

```diff
@@ -106,6 +106,11 @@ struct Paint {
   std::shared_ptr<Contents> CreateContentsForGeometry(
       std::unique_ptr<Geometry> geometry) const;
 
+  /// @brief   Whether this paint has a filter that can apply opacity
+  ///
+  ///          Only color filters can absorb opacity currently.
+  bool HasFilters() const;
```

Comment:
This just checks for a color filter, not any filter. So perhaps name `HasColorFilter` instead?

Reply:
Done